openmetadata-ui/src/main/resources/ui/src/components/common/FormBuilder/FormBuilder.tsx

```diff
@@ -12,27 +12,26 @@
  */
 
 import { CheckOutlined } from '@ant-design/icons';
-import Form from '@rjsf/antd';
-import CoreForm, { AjvError, FormProps, IChangeEvent } from '@rjsf/core';
-import validateFormData from '@rjsf/core/lib/validate';
+import Form, { FormProps, IChangeEvent } from '@rjsf/core';
 import { Button as AntDButton } from 'antd';
 import classNames from 'classnames';
+import { ArrayFieldTemplate } from 'components/JSONSchemaTemplate/ArrayFieldTemplate';
 import { customFields } from 'components/JSONSchemaTemplate/CustomFields';
+import { ObjectFieldTemplate } from 'components/JSONSchemaTemplate/ObjectFieldTemplate';
 import { ServiceCategory } from 'enums/service.enum';
 import { useAirflowStatus } from 'hooks/useAirflowStatus';
 import { t } from 'i18next';
-import { isEmpty, isUndefined, startCase } from 'lodash';
+import { isEmpty, isUndefined } from 'lodash';
 import { LoadingState } from 'Models';
 import React, { FunctionComponent, useEffect, useRef, useState } from 'react';
 import { getPipelineServiceHostIp } from 'rest/ingestionPipelineAPI';
+import { transformErrors } from 'utils/formUtils';
 import { ConfigData } from '../../../interface/service.interface';
 import { formatFormDataForRender } from '../../../utils/JSONSchemaFormUtils';
-import { ArrayFieldTemplate } from '../../JSONSchemaTemplate/ArrayFieldTemplate';
-import { ObjectFieldTemplate } from '../../JSONSchemaTemplate/ObjectFieldTemplate';
 import Loader from '../../Loader/Loader';
 import TestConnection from '../TestConnection/TestConnection';
 
-interface Props extends FormProps<ConfigData> {
+interface Props extends FormProps {
```

Comment:
Don't we need to pass `ConfigData` in the `FormProps` generic type?

Reply:
We I have removed onFocus prop which was using it